src/robot/libraries/BuiltIn.py

```diff
@@ -2199,6 +2202,7 @@ def wait_until_keyword_succeeds(self, retry, retry_interval, name, *args):
         Examples:
         | Wait Until Keyword Succeeds | 2 min | 5 sec | My keyword | argument |
         | ${result} = | Wait Until Keyword Succeeds | 3x | 200ms | My keyword |
+        | ${result} = | Wait Until Keyword Succeeds | 3x | strict: 200ms | My keyword |
```

Comment:
Good to add an example!

There should be a note like this at the end of the docstring:
> Support for "strict" retry interval is new in Robot Framework 4.1.